index.html

```diff
@@ -12598,7 +12598,7 @@ <h2>Relationship to RDF</h2>
     <li class="changed">As an extension to the <a data-cite="RDF11-CONCEPTS#data-model">RDF data model</a>,
       <em>typed literals</em>, where the datatype is `xsd:string`, or <a>language-tagged strings</a>
       MAY include a <a>text direction</a>.
-      As there is not yet any standardized mechanism for representing the text direction
+      As there is currently no standardized mechanism for representing the <a>text direction</a>
       of <a>RDF literals</a>, the JSON-LD to RDF transformation can loose this direction.
```

Comment:
loose -> lose?

Or perhaps this should read more like ".. the JSON-LD to RDF transformation generally loses the base direction..."

Reply:
Fixed in 9b1e522554ac86b8079f21b10ca254a2c478ec2a.